2-Webpack/webpack.config.js

```diff
@@ -0,0 +1,23 @@
+const path = require('path');
+const CopyWebpackPlugin = require('copy-webpack-plugin')
+
+module.exports={
+    mode:"development",
```

Comment:
As we discussed today, it actually makes more sense to have `production` as the default mode when bundling. `development` is suitable for dev-server settings